README.md

```diff
@@ -1 +1,38 @@
-# Group_E_Simulation_Life_Cycle
\ No newline at end of file
+# Group_E_Simulation_Life_Cycle
+Françoise Blanc
+Roman Cardenas Rodriguez
+Ghida Dandan
+Aidan Fahlman
+Bruno St-Aubin
+
+## Description
+Need to Develop this Section
+
+
+## Software Requirements
+1. Users must be able to provide geospatial data as a source for the model generation workflow:
+	1. All geometric primitives (point, line and polygon)
+	2. Spatial network data  
+	3. In all common geospatial data formats (shapefile, geopackages, csv, geojson, etc.)
+2. Users must be able to provide attribute data in common formats (csv, excel, json) for the model generation process.
+3. Users must be able to provide a workflow file that contains the steps for automated model generation
+4. The system must automatically execute the workflow to convert geospatial data into a Cadmium model configuration file.
+	1. Map geospatial features onto simulation models from an existing library of parametric models
+	2. Establish neighborhoods or couplings through spatial analysis of the geospatial data
+	3. The file output must be a json file properly formatted according to Roman’s model generation process
+5. The system must support common spatial analysis functions to establish neighborhood and parameters:
+	1. Topological relationships (intersection, contains, within, touches, etc.)
+	2. Network distance analysis
+	3. Volume analysis
+	4. Spatial buffers
+	5. Voronoi diagrams
+	6. Geospatial statistics
+6. The system must support common SQL like function to determine model parameters
```

Comment:
I find this requirement too technical. I suggest we remove it

Reply:
Are you referring to requirement 6? If so, I agree that we should remove it as it mainly refers to the implementation.

Reply:
Just to clarify, the idea was that the system, in addition to offering spatial analysis capability, should also provide standard querying capabilities to build simulation models. Maybe the wording was not clear. But these are two different concepts in GIS. One relies on geometry the other, on descriptive attributes.

Reply:
I'm resolving this conversation